APE24.rst

```diff
@@ -0,0 +1,443 @@
+|                          | 2 reviewers (1 can be       |                            |
+|                          | non-domain specific)        |                            |
++--------------------------+-----------------------------+----------------------------+
+| Editorial team           | `pyOpenSci Editorial Board`_| 2 editors that rotate, see |
+|                          |                             | `Astropy team roles`_      |
++--------------------------+-----------------------------+----------------------------+
+| Review style             | Open review                 | Reviewer is anonymous by   |
+|                          |                             | default                    |
++--------------------------+-----------------------------+----------------------------+
+| Post-review checks       | We will be creating a set of| Astropy has static badges  |
+|                          | checks to see if the package| that need to be manually   |
+|                          | is still maintained and in  | updated.                   |
+|                          | what state. We might be able|                            |
+|                          | to work with Astropy (and   |                            |
+|                          | other communities) on what  |                            |
+|                          | this looks like.            |                            |
++--------------------------+-----------------------------+----------------------------+
+
+.. _pyOpenSci Editorial Board: https://www.pyopensci.org/about-peer-review/#our-editorial-board
+.. _Astropy team roles: https://www.astropy.org/team#roles
+
+As of July 2023, Astropy had the following `pre-APE 24 Affiliated Package review guidelines`_
+and followed this `pre-APE 24 Affiliated Package review process, with templates`_.
+
+.. _pre-APE 24 Affiliated Package review guidelines: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_guidelines.md
+.. _pre-APE 24 Affiliated Package review process, with templates: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_process.md
+
+Meanwhile, pyOpenSci has its own
+`submission guide for package authors <https://www.pyopensci.org/software-peer-review/how-to/author-guide.html>`_,
+`accompanying GitHub issue templates <https://github.com/pyOpenSci/software-submission/issues/new/choose>`_, and
+`listing of accepted packages <https://www.pyopensci.org/python-packages.html#our-accepted-python-open-source-packages>`_.
+As of July 2023, they are in the process of revamping the submission form to handle
+collaborations with different communities (e.g., Astropy), including but not limited to:
+
+* creating a community-specific label for filtering and link to current reviews
+  for that community; and
+* customizing the editorial bot to add community-specific tags.
```

Comment:
```suggestion
collaborations with different communities (e.g., Astropy). There are also:

* creating a community-specific label for package filtering and link to current reviews
  for that community on their website package listing; and
* customizing the editorial bot to add community-specific tags.
```

Reply:
i'm not sure if this is quite right but

there are two buckets of activities here

1. updating the website to display packages and have astropy filtering 
2. updating the submission template so a potentially astropy affiliated package can select that option for the actual review. 
3. we can totally add an astropy label to github - that is easy

we don't have a bot YET - BUT i'll be working on that with Arfon from JOSS starting in the late fall to automate some of our review steps :) Right now the editors would do this :)

Reply:
I updated the bullet points. I hope they reflect all your comments here.

Reply:
this looks good!